projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrService.java

```diff
@@ -623,11 +623,8 @@ public Response getContainer(
       checkContainerAccessibility(apiKey, containerName);
 
       Container container = Main.getWorkMgr().getContainer(containerDir);
-      container.setName(containerName);
-      BatfishObjectMapper mapper = new BatfishObjectMapper();
-      String containerString = mapper.writeValueAsString(container);
 
-      return Response.ok(containerString, MediaType.APPLICATION_JSON).build();
+      return Response.ok(container).build();
```

Comment:
Use jackson to convert it into a Json String now, need to determine how many information to be returned in future.